Formula/f/fwupd.rb

```diff
@@ -0,0 +1,106 @@
+class Fwupd < Formula
+  include Language::Python::Shebang
+  include Language::Python::Virtualenv
+
+  desc "Firmware update daemon"
+  homepage "https://github.com/fwupd/fwupd"
+  url "https://github.com/fwupd/fwupd/releases/download/1.9.5/fwupd-1.9.5.tar.xz"
+  sha256 "201ec20538044749c96dbff708e52f7d79038de787b90a2762e76793080aa37c"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/fwupd/fwupd.git", branch: "main"
+
+  depends_on "gi-docgen" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python-markupsafe" => :build
+  depends_on "python@3.11" => :build
+  depends_on "vala" => :build
+  depends_on "gcab"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gnutls"
+  depends_on "json-glib"
+  depends_on "libcbor"
+  depends_on "libgusb"
+  depends_on "libjcat"
+  depends_on "libxmlb"
+  depends_on "protobuf-c"
+  uses_from_macos "curl"
+  uses_from_macos "libarchive"
+  uses_from_macos "sqlite"
+
+  resource "jinja2" do
+    url "https://files.pythonhosted.org/packages/7a/ff/75c28576a1d900e87eb6335b063fab47a8ef3c8b4d88524c4bf78f670cce/Jinja2-3.1.2.tar.gz"
+    sha256 "31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852"
+  end
+
+  def python3
+    "python3.11"
+  end
+
+  def install
+    rewrite_shebang detected_python_shebang, "generate-build/generate-dbus-interface.py"
+    rewrite_shebang detected_python_shebang, "generate-build/generate-metainfo.py"
+    rewrite_shebang detected_python_shebang, "generate-build/generate-plugins-header.py"
+    rewrite_shebang detected_python_shebang, "generate-build/generate-quirk-builtin.py"
+    rewrite_shebang detected_python_shebang, "generate-build/generate-version-script.py"
+
+    # Set up Jinja2
+    venv_root = libexec/"venv"
```

Comment:
this venv isn't needed at runtime right?
```suggestion
    venv_root = buildpath/"venv"
```

Reply:
Correct, I've fixed this up in the original commit. Good catch, thanks!